driver-opentelemetry/src/main/java/com/datastax/driver/opentelemetry/OpenTelemetryTracingInfo.java

```diff
@@ -1,5 +1,7 @@
 package com.datastax.driver.opentelemetry;
 
+import static com.datastax.driver.opentelemetry.PrecisionLevel.FULL;
```

Comment:
I think you should do 
```
import com.datastax.driver.opentelemetry.PrecisionLevel;
```
and then use ```PrecisionLevel.FULL``` instead of ```FULL```.